--- browser/app/Makefile.in.orig	2014-12-01 19:16:51.905874589 +0000
+++ browser/app/Makefile.in	2014-12-01 19:21:18.710916512 +0000
@@ -19,7 +19,7 @@
   $(NULL)
 
 ifdef LIBXUL_SDK #{
-PREF_JS_EXPORTS += $(srcdir)/profile/channel-prefs.js
+PREF_JS_EXPORTS += $(srcdir)/profile/channel-prefs.js $(srcdir)/profile/kde.js
 endif #} LIBXUL_SDK
 
 # Build a binary bootstrapping with XRE_main
@@ -82,6 +82,9 @@
 libs:: $(srcdir)/profile/channel-prefs.js
 	$(NSINSTALL) -D $(DIST)/bin/defaults/pref
 	$(call py_action,preprocessor,-Fsubstitution $(PREF_PPFLAGS) $(ACDEFINES) $^ -o $(DIST)/bin/defaults/pref/channel-prefs.js)
+
+libs:: $(srcdir)/profile/kde.js
+	$(INSTALL) $(IFLAGS1) $^ $(DIST)/bin/defaults/pref
 endif
 
 libs:: $(srcdir)/blocklist.xml
