--- kmozillahelper/CMakeLists.txt.orig	2013-05-06 21:13:05.500000000 +0000
+++ kmozillahelper/CMakeLists.txt	2013-05-06 21:13:33.539000000 +0000
@@ -17,6 +17,6 @@
 target_link_libraries( kmozillahelper ${KDE4_KFILE_LIBS} )
 
 # TODO hardcoded path?
-install( TARGETS kmozillahelper DESTINATION /usr/lib/mozilla/ )
+install( TARGETS kmozillahelper DESTINATION /extra/usr/lib/mozilla/ )
 
 install( FILES kmozillahelper.notifyrc  DESTINATION  ${DATA_INSTALL_DIR}/kmozillahelper )
