diff -U 3 -B -d -t -r -- old/kcm_adobe_flash_player.desktop new/kcm_adobe_flash_player.desktop
--- old/kcm_adobe_flash_player.desktop	2014-05-28 07:54:24.000000000 +0800
+++ new/kcm_adobe_flash_player.desktop	2014-06-12 08:31:54.234504563 +0800
@@ -8,7 +8,7 @@
 X-KDE-Library=kcm_adobe_flash_player
 X-KDE-ParentApp=kcontrol
 
-X-KDE-System-Settings-Parent-Category=personal
+X-KDE-System-Settings-Parent-Category=network-and-connectivity
 X-KDE-Weight=10
 
 Name=Adobe Flash Player

